### PR TITLE
Lesson 04 - three tasks

### PR DESCRIPTION
**Pull request by karinanartowska**

### CODE REVIEW

**Comment by karinanartowska** [comment]
@gradopaw please review :)

File: students/nartowska_karina/lesson_04_unit_testing/test_for_fibonacci_numbers.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import unittest
+
+
+def fib(n):
```
Comment (gradopaw) [inline students/nartowska_karina/lesson_04_unit_testing/test_for_fibonacci_numbers.py]: Lines from 4 to 21 are not needed - the aim of this task is to test only fib function:) You can just import fib function from the fibonacci_numbers module from lesson_03_functions. To do it:
- mark your folder (nartowska_karina) as src folder (go to PyCharm-> right click on the karina_nartowska folder -> Mark Directory as -> choose source root
- Add empty __init__.py file in the lesson_03_functions to treat this directory as containing packages (have a look at https://docs.python.org/3/tutorial/modules.html#packages)
- now in the test_for_fibonacci_numbers,py you should have the possibility to import function you would like to test (from lesson_03_functions.fibonacci_numbers import fib)

File: students/nartowska_karina/lesson_04_unit_testing/unit_tests_for_distance.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import unittest
+from math import sqrt
+
+
+def distance(x1, y1, x2, y2):
```
Comment (gradopaw) [inline students/nartowska_karina/lesson_04_unit_testing/unit_tests_for_distance.py]: Look at my comment to test_for_fibonacci_numbers.py. Before you will import distance function you should add a fix to it (function should return value:))

File: students/nartowska_karina/lesson_04_unit_testing/unit_tests_for_collatz.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import unittest
+
+
+def collatz(number):
```
Comment (gradopaw) [inline students/nartowska_karina/lesson_04_unit_testing/unit_tests_for_collatz.py]: Look at my comment to the previous file.

File: students/nartowska_karina/lesson_04_unit_testing/test_for_fibonacci_numbers.py (hunk 1 above)
Comment (gradopaw) [reply students/nartowska_karina/lesson_04_unit_testing/test_for_fibonacci_numbers.py]: the proper name of the empty file you should add "__init__.py". Don't forget to push this file on GitHub as well

**Comment by karinanartowska** [comment]
@gradopaw please review :)

**Pull request merged**